docs/code/MlNetCookBook.md

```diff
@@ -578,6 +578,60 @@ var biases = modelParameters.GetBiases();
 
 ```
 
+## How do I get a model's weights to look at the global feature importance?
+The below snippet shows how to get a model's weights to help determine the feature importance of the model for a linear model.
+
+```csharp
+var linearModel = model.LastTransformer.Model;
+
+var weights = linearModel.Weights;
+```
+
+The below snipper shows how to get the weights for a fast tree model.
+
+```csharp
+var treeModel = model.LastTransformer.Model;
+
+var weights = new VBuffer<float>();
+treeModel.GetFeatureWeights(ref weights);
+```
+
+## How do I look at the global feature importance?
+The below snippet shows how to get a glimpse of the the feature importance, or how much each feature impacts the performance of the model. It also outputs the difference in root mean squared for each feature as though the feature were replaced with a random value.
+
+```csharp
+var transformedData = model.Transform(data);
+
+var featureImportance = context.Regression.PermutationFeatureImportance(model.LastTransformer, transformedData);
+
+foreach (var metricsStatistics in featureImportance)
+{
+    Console.WriteLine($"Feature I: Difference in RMS - {metricsStatistics.Rms.Mean}");
+}
+```
+
+## How do I look at the local feature importance per example?
+The below snippet shows how to get feature importance for each example of data.
```

Comment:
>feature importance for each example of data [](start = 35, length = 43)

Can you link to the appropriate place in docs for more information for all of these? Maybe we don't actually need to go into major details here.

Reply:
The best doc I could find is [this one](https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/determine-global-feature-importance-in-model). Is this ok to link to in each of these sections or would there be a doc for each of these?

Reply:
Oh, I was thinking we could link to the code samples in the repo. But this is a moving target, so let's revisit later.

Reply:
Sounds good! Apologies for the misunderstanding. Was there anything else I missed for the PR? Just making sure no one is waiting for me to make more updates 😄